docs/using.md

```diff
@@ -569,6 +570,13 @@ It is often necessary to use Genny to operate with large amounts of data which w
 
 Value generators are not a builtin feature of Genny, but must be integrated by each Actor for the configuration values that accept them. For examples of using value generators, see [./src/workloads/docs/Generators.yml](../src/workloads/docs/Generators.yml). To integrate generators into an Actor, use the [DocumentGenerator](../src/value_generators/include/value_generators/DocumentGenerator.hpp) with the yaml node you intend to generate documents from. (And see [here](#org7e6c6bd) for more details on creating an Actor in the first place.)
 
+Some Value Generators, like `ChooseFromDataset`, can read a dataset from the disk that store the values to choose from. Some example datasets are stored in [./src/workloads/datasets](../src/workloads/datasets). Four are included in the repo:
+
+- **airport_codes.txt:** includes all airport codes
```

Comment:
Could you add a README.md file to the datasets directory. Maybe move this content there, and replace with a link.

Reply:
Added the README.md and a link to in the usage.md